src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -33432,5 +33432,121 @@ class C23_2 : I2<C23_2>, I3<C23_2>
                     Diagnostic(ErrorCode.ERR_DuplicateConversionInClass, "int").WithArguments("C23_2").WithLocation(80, 45)
                 );
         }
+
+        [Fact]
```

Comment:
Consider putting a `WorkItem` attribute with a link to the bug these are fixing.